### ISSUE TITLE
Failed to load https://api.twitter.com/1.1/statuse

### ISSUE THREAD

**Issue by Mas99** [RipMeApp/ripme#1953]
* Ripme version: 1.7.95
* Java version:  Version 8 update 241 
* Operating system:  windowns

* Exact URL you were trying to rip when the problem occurred: https://twitter.com/xxx
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Should rip all the images and videos' including all RT's etc

## Actual Behavior

Got this error below. I keep getting this error everyday 

Failed to load https://api.twitter.com/1.1/statuses/user_timeline.json?screen_name=xxx&include_entities=true&exclude_replies=true&trim_user=true&count=200&tweet_mode=extended: Status Code 401. You might be able to circumvent this error by setting cookies for this domain

**Comment by SysAdminDoc** [comment]
This is a big one. I can't get it to work either.

**Comment by 89z** [comment]
please provide example `screen_name`

**Comment by carfesh** [comment]
I don't think that a screen name is needed; I'm seeing this issue as well, and it is highly sporadic and not related to particular accounts. I will sometimes have this 10 times in succession, and on the 11th it will work; it seems to vary wildly. I can only guess it may be related to the availability of the Twitter API, or to what was done from my IP before, I don't know.

It usually happens on the initial connection for a newly added account, not once ripping has commenced - maybe some kind of retry with a timed backoff might help, it's hard to say.

**Comment by LudovicoSforza** [comment]
Maybe because you're trying to download from protected account?

**Comment by Mas99** [comment]
> Maybe because you're trying to download from protected account?

No, it is public account
